models/members.js

```diff
@@ -13,21 +13,27 @@ const userModel = firestore.collection('users')
 
 const fetchMembers = async () => {
   try {
-    const snapshot = await userModel.where('isMember', '==', true).get()
+    const snapshot = await userModel.get()
 
     const allMembers = []
 
     if (!snapshot.empty) {
       snapshot.forEach((doc) => {
-        allMembers.push({
+        const memberData = doc.data()
+        const curatedMemberData = {
           id: doc.id,
-          ...doc.data(),
+          ...memberData,
           tokens: undefined,
           phone: undefined,
           email: undefined
-        })
-      }
-      )
+        }
+        if (memberData.roles && memberData.roles.member) {
+          allMembers.isMember = true
+        } else {
+          allMembers.isMember = false
+        }
```

Comment:
Nit:
```
          allMembers.isMember = !!(memberData.roles && memberData.roles.member)
```

Reply:
shouldn't this be `curatedMemberData.isMember`?